src/styles/variables/shadows.js

```diff
@@ -0,0 +1,6 @@
+export const boxShadows = {
+  articleCard: '16px 16px 32px rgba(0, 0, 0, 0.05)',
```

Comment:
Keep this color combinations consistent use hex colors

Reply:
This is too much for box shadow. It should be smooth and subtle https://www.crazyegg.com/blog/css3-box-shadow/